gettsim/pre_processing/piecewise_functions.py

```diff
@@ -28,27 +28,42 @@ def piecewise_polynomial(
         The value of `x` under the piecewise function.
 
     """
+    # If no individual is transferred, we return an empty series
+    if x.empty:
```

Comment:
Necessary?

Reply:
Yes. E.g. for `eink_anr_frei`, the function is called for individuals with kids and without. If we have in one of the groups no person, then it is an empty call and can not be processed otherwise!

Reply:
This belongs into the calling function then. Clearer and should be faster.

`df.loc[n_kinder > 0] =`

Reply:
Yes, we do restrict the call [here](https://github.com/iza-institute-of-labor-economics/gettsim/blob/defb9a869520c4373646cf27649bf7c2d23edd2a/gettsim/benefits/arbeitsl_geld_2/eink_anr_frei.py#L22). But maybe there are no individuals in one of the groups!

Reply:
Seems weird. Why would it fail then? Again, add to that issue, please, there are more pressing things right now.

Reply:
Okay. I put in the issue and delete it for now!

Reply:
Only delete if it is certain it won't fail.

Reply:
Yes I am certain!

Reply:
Sorry. I deleted the one below! This is still there and has to be!